ansible/library/generate_golden_config_db.py

```diff
@@ -383,6 +383,41 @@ def update_dns_config(self, config):
         else:
             return config
 
+    def generate_default_golden_config_db(self):
+        rc, out, err = self.module.run_command("sonic-cfggen -H -m -j /etc/sonic/init_cfg.json --print-data")
+        if rc != 0:
+            self.module.fail_json(msg="Failed to get config from minigraph: {}".format(err))
+
+        # Generate config table from init_cfg.ini
+        ori_config_db = json.loads(out)
+
+        golden_config_db = {}
+        if "DEVICE_METADATA" in ori_config_db:
+            golden_config_db["DEVICE_METADATA"] = ori_config_db["DEVICE_METADATA"]
+
+        return json.dumps(golden_config_db, indent=4)
+
+    def update_zmq_config(self, config):
+        ori_config_db = json.loads(config)
+        if "DEVICE_METADATA" not in ori_config_db:
+            ori_config_db["DEVICE_METADATA"] = {}
+        if "localhost" not in ori_config_db["DEVICE_METADATA"]:
+            ori_config_db["DEVICE_METADATA"]["localhost"] = {}
+
+        # multiasic test failed because this field is 'None', which break yang validation.
```

Comment:
> break yang validation [](http://example.com/codeflow?start=67&length=22)

Should we fix multiasic feature fist?

Reply:
Will create a sonic-buildimage issue for docker_routing_config_mode
Also create a PR to only enable orch_northbond_route_zmq_enabled feature in BGP test.

Reply:
Fixed, seems the multiasic feature fixed in image.